packages/backend-deployer/src/cdk_error_mapper.ts

```diff
@@ -387,6 +387,15 @@ export class CdkErrorMapper {
       errorName: 'CFNUpdateNotSupportedError',
       classification: 'ERROR',
     },
+    {
+      errorRegex: /connect ENOMEM (?<remoteAddress>\d+\.\d+\.\d+\.\d+).*/,
```

Comment:
could this remove address be a domain name?

Reply:
I have not seen any example where this would be a domain name.

This error comes from `internalConnect	node:net` which is presumably after DNS resolution and node is opening a socket given an ip address.